src/ExchangeContainer/ExchangeContainer.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from 'react'
+import { Route } from 'react-router-dom'
+import { Link, NavLink } from 'react-router-dom';
+import './ExchangeContainer.scss';
+import CurrencyCard from '../CurrencyCard/CurrencyCard.js'
+import Bookmarked from '../Bookmarked/Bookmarked.js'
+
+const ExchangeContainer = () => {
+  return (
+    <section className='container'>
+      <h1 >This is the container for the currency cards</h1>
+      {/* <CurrencyCard />
+      <Bookmarked />
```

Comment:
I think I actually need to put Bookmarked in App's render and put a navLink in ExchangeContainer